### DISCUSSION TITLE
Setup Private Email DNS

### DISCUSSION THREAD

**Discussion by rajatsonibkn2002** [vercel/next.js#51746] [Help] [answered]
### Summary

I bought my domain from Namecheap and added it to my Vercel application. I have bought emails from Namecheap, and they are asking me to get my email DNS added from Vercel. How can I do that?

**Comment by MrAlaminH** [comment, marked as answer]
Here are the steps on how to add your email DNS from Namecheap to Vercel:

1. Log in to your Namecheap account.
2. Click on the **Domains** tab.
3. Click on the name of your domain.
4. Click on the **Advanced DNS** tab.
5. Scroll down to the **Mail Settings** section.
6. Click on the **Custom MX** button.
7. Enter the MX records provided by Namecheap.
8. Click on the **Save** button.

Once you have added the MX records, it may take some time for the changes to propagate. You can check the status of the propagation by visiting the DNS Checker website: https://dnschecker.org/.

Here are the steps on how to add a custom domain to your Vercel project:

1. Go to the Vercel dashboard and click on the project that you want to add the domain to.
2. Click on the **Settings** tab.
3. Click on the **Domains** section.
4. Click on the **Add** button.
5. Enter the domain name that you want to add.
6. Click on the **Add** button.

It may take some time for the DNS changes to propagate. Once the changes have propagated, you should be able to send and receive emails using your Namecheap email address.

Here are some additional resources that you may find helpful:

How to Set Up Email with Your Vercel Domain: https://vercel.com/guides/using-email-with-your-vercel-domain
How to Add Custom MX Records in Namecheap: https://www.namecheap.com/support/knowledgebase/article.aspx/322/2237/how-can-i-set-up-mx-records-required-for-mail-service/

**Reply by rajatsonibkn2002** [reply]
Yes, it worked for me. Thanks!

**Comment by gmhislop** [comment]
This worked for me: 

Within the Vercel platform, follow these steps to configure your DNS records:

1. Navigate to the Vercel dashboard.

2. Click on the "Domain" section.

3. Fill in the DNS record information as required.

For additional details on Namecheap DNS configurations, you can refer to this guide: [Namecheap Private Email Records for Domains with Third-Party DNS](https://www.namecheap.com/support/knowledgebase/article.aspx/1340/2176/namecheap-private-email-records-for-domains-with-thirdparty-dns/).

**Comment by msidqi** [comment]
if you're using Custom DNS for a domain bought in namecheap and Vercel as third-party:

you need to follow to add MX and TXT records described in this link to your domain's settings in vercel platform https://www.namecheap.com/support/knowledgebase/article.aspx/1340/2176/namecheap-private-email-records-for-domains-with-thirdparty-dns/

**Reply by Rupsnigdha** [reply]
Quick question, where do I enter the host name in the Vercel DNS Configuration?
![image](https://github.com/vercel/next.js/assets/83987752/3ebdccae-d389-482d-b491-2e237bad8f10)
![image](https://github.com/vercel/next.js/assets/83987752/17d66ceb-6866-4a0e-a51b-b2d64b02f90f)
Could you please guide me on which field goes where?
